modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -299,7 +301,9 @@ public ObjectDefinition putObjectDefinition(
 				_objectDefinitionService.getObjectDefinition(
 					objectDefinitionId);
 
-		if (!serviceBuilderObjectDefinition.isApproved()) {
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-164278")) &&
+			!serviceBuilderObjectDefinition.isApproved()) {
+
```

Comment:
@murilostodolni not sure if we need this feature flag here, since `ObjectFieldUtil.addListTypeDefinition` is already under feature flag.

Reply:
Perfect!